File: Examples/test-suite/matlab/enums_runme.m
import .*
enums

enums.bar2(1)
enums.bar3(1)
enums.bar1(1)

if (enums.cvar.enumInstance ~= 2)
  error
end

if (enums.cvar.Slap ~= 10)
  error
end

if (enums.cvar.Mine ~= 11)
  error
end

if (enums.cvar.Thigh ~= 12)
  error
end

File: Examples/test-suite/javascript/enums_runme.js

var enums = require("enums");

/* await */(enums.bar2(1));
/* await */(enums.bar3(1));
/* await */(enums.bar1(1));

if (enums.enumInstance != 2) {
    throw new Error;
}

if (enums.Slap != 10) {
    throw new Error;
}

if (enums.Mine != 11) {
    throw new Error;
}

if (enums.Thigh != 12) {
    throw new Error;
}
